packages/terra-paginator/src/ProgressivePaginator.jsx

```diff
@@ -230,63 +230,69 @@ class ProgressivePaginator extends React.Component {
       <>
         <div>
           {
-            (totalCount) && (
-              <PaginatorButton
-                ariaDisabled={selectedPage === 1}
-                ariaLabel={intl.formatMessage({ id: 'Terra.paginator.first' })}
-                className={cx(['nav-link', selectedPage === 1 ? 'is-disabled' : null])}
-                disabled={selectedPage === 1}
-                onClick={this.handlePageChange(1)}
-                onKeyDown={this.handlePageChange(1)}
-              >
-                {intl.formatMessage({ id: 'Terra.paginator.first' })}
-              </PaginatorButton>
-            )
+            intl.formatMessage({ id: messageId }, messageAttributes)
```

Comment:
Do we need to have a left padding style?

Reply:
replied on the other comment: https://github.com/cerner/terra-core/pull/3897#issuecomment-1697684046